application/helpers/template_helper.php

```
@@ -86,15 +86,15 @@ function parse_template($object, $body)
                     if (preg_match('/ip_cf_(\d.*)/', $var, $cf_id)) {
                         // Get the custom field
                         $CI = & get_instance();
-                        $CI->load->model('custom_fields/mdl_custom_fields');
+                        $CI->load->model('custom_fields/custom_field');
                         $cf = $CI->mdl_custom_fields->get_by_id($cf_id[1]);
```

PHPMD (2.15.0)

Avoid variables with short names like $cf. Configured minimum length is 3. (undefined)

(ShortVariable)